src/introspection/getSchemaFromData.js

```diff
@@ -140,16 +140,25 @@ export default (data) => {
                 },
                 {}
             );
+            const createFields = Object.keys(typeFields).reduce(
+              (f, fieldName) => {
```

Comment:
the `f` variable is cryptic. Perhaps rename it to `acc` for accumulator?

Reply:
Already applied to object destructuring approach.